staking/staking_hub_test.go

```diff
@@ -0,0 +1,262 @@
+package staking
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/minimal/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func EthToWei(ethValue int64) *big.Int {
+	return new(big.Int).Mul(
+		big.NewInt(ethValue),
+		new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil))
+}
```

Comment:
I guess you couldn't import the one from `e2e/framework/helper.go`?

Reply:
Maybe pull that into some outer directory, so that we don't have two methods with the same code?

Reply:
Pulling this out to a specific helper directory, so it's visible wherever we need it 👍